src/app/components/Field/Field.tsx

```diff
@@ -1,19 +1,155 @@
+import { useEffect, useRef, useState } from 'react'
+import { useWindowSize } from '@react-hook/window-size'
 import Box, { BoxProps } from '@mui/material/Box'
+import Grid from '@mui/material/Grid'
+import useTheme from '@mui/material/styles/useTheme'
 
+import { PlayedCrop } from '../PlayedCrop'
+import * as cards from '../../../game/cards'
 import { lookup } from '../../../game/services/Lookup'
 import { IGame, IPlayer } from '../../../game/types'
+import { isCardId } from '../../../game/types/guards'
+import { UnimplementedError } from '../../../game/services/Rules/errors'
+import { SELECTED_CARD_ELEVATION } from '../../../game/config'
+
+const deselectedIdx = -1
+const selectedCardYOffset = -25
 
 export interface FieldProps extends BoxProps {
   game: IGame
   playerId: IPlayer['id']
 }
 
+export const rotationTransform = 'rotate(180deg)'
+
 export const Field = ({ playerId, game, ...rest }: FieldProps) => {
   const player = lookup.getPlayer(game, playerId)
+  const isSessionOwnerPlayer = playerId === game.sessionOwnerPlayerId
+
+  const containerRef = useRef<HTMLDivElement>()
+  const theme = useTheme()
+  const [selectedCardIdx, setSelectedCardIdx] = useState(deselectedIdx)
+  const [selectedCardTransform, setSelectedCardTransform] = useState('')
+  const [windowWidth, windowHeight] = useWindowSize()
+
+  const centerX = windowWidth / 2
+  const centerY = windowHeight / 2
+
+  const resetSelectedCard = () => {
+    setSelectedCardIdx(deselectedIdx)
+
+    if (document.activeElement instanceof HTMLElement) {
+      document.activeElement.blur()
+    }
+  }
+
+  useEffect(() => {
+    const handleWindowResize = () => {
```

Comment:
not necessarily blocking, but i prefer to `debounce` or `throttle` any window resize handlers. i'd probably debounce in this specific case

Reply:
Great idea! This is done in https://github.com/jeremyckahn/farmhand-shuffle/commit/6fc2588ba88e02bbbcad06008fe320271d9e4c04.